4-WebApp-your-API/4-1-MyOrg/AppCreationScripts/sample.json

```diff
@@ -108,10 +120,12 @@
       "RequiredResourcesAccess": [
         {
           "Resource": "service",
-          "DelegatedPermissions": [ "ToDoList.Read", "ToDoList.Write" ]
+          "DelegatedPermissions": [ "ToDoList.Read", "ToDoList.ReadWrite" ]
         }
       ],
       "ManualSteps": [],
+      "EnableAccessTokenIssuance": "true",
+      "EnableIdTokenIssuance": "false",
```

Comment:
>"EnableIdTokenIssuance": "false", [](http://example.com/codeflow?start=6&length=33)

isn't this the default?

Reply:
yes, the false is default, but for my opinion the option should be exposed for educational reasons. If it not exposed then a developer won't know about it's existence. The sample will be the reference for others.

Reply:
No, actually this setting is only for apps that use implicit flow, and this is outright wrong.
Unless the web app is a SPA, this setting should not be bothered with please.